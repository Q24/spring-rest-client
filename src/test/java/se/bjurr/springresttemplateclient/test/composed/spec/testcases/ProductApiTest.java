package se.bjurr.springresttemplateclient.test.composed.spec.testcases;

import se.bjurr.springresttemplateclient.test.composed.spec.api.ProductApi;
import se.bjurr.springresttemplateclient.test.utils.BaseApiTest;

public class ProductApiTest extends BaseApiTest<ProductApi> {

  @Override
  public Class<ProductApi> getSutClass() {
    return ProductApi.class;
  }
}
