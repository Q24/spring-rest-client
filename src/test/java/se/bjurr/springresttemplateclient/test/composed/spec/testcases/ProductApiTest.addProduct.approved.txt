[ {
  "url" : "http://localhost:X/product",
  "method" : "POST",
  "headers" : {
    "Accept" : "application/json, application/*+json",
    "Connection" : "keep-alive",
    "Content-Length" : "8",
    "Content-Type" : "application/json;charset=UTF-8"
  },
  "cookies" : { },
  "body" : "{\"id\":1}"
} ]